bin/install-if-deps-outdated.js

```diff
@@ -24,7 +24,7 @@ const needsInstall = () => {
 };
 
 if ( needsInstall() ) {
-	const installResult = spawnSync( 'npm', [ 'install' ], {
+	const installResult = spawnSync( 'lerna', [ 'bootstrap', '--hoist' ], {
```

Comment:
The [Getting started](https://github.com/Automattic/wp-calypso/blob/master/README.md#getting-started) section on the README says:

> 1.	Make sure you have `git`, `node`, and `npm` installed.
> 2.	Clone this repository locally.
> 3.	Add `127.0.0.1 calypso.localhost` to your local `hosts` file.
> 4.	Execute `npm start` or `npm run dashboard` (for a more visually-oriented interface) from the root directory of the repository.
> 5.	Open [`calypso.localhost:3000`](http://calypso.localhost:3000/) in your browser.

With this change to `lerna`, this is no longer accurate. `lerna` must either be installed globally (yuck), or we'll need to include add an `npm install` step to the getting started list. That would be unfortunate as it's what we're looking to avoid with these `install-if-*` scripts.

Maybe we could add another magical install script just to check whether `lerna` is available, and `npm i lerna` if not.

If you haven't installed `lerna` and try to `npm start`, 💥  (and not in a good way 🙂).

I think this is a blocker as it's a barrier to entry on the project.